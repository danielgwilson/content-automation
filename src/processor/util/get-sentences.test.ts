import { getSentences } from "./get-sentences";

describe("Sentences", () => {
  it("Gets sentences without removing line break", () => {
    const text = "Sentence 1.\n\nSentence 2.";
    const sentences = getSentences(text);
    expect(sentences).toMatchSnapshot();
  });
});
