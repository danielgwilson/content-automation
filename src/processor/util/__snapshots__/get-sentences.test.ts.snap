// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sentences Gets sentences without removing line break 1`] = `
Array [
  "Sentence 1.
",
  "
Sentence 2.",
]
`;
